.github/workflows/chromatic.yml

```diff
@@ -0,0 +1,19 @@
+name: "Chromatic"
+on: push
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Use Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: "10.16.3"
+      - name: Install dependencies
+        run: yarn setup
+      - uses: chromaui/action@v1
+        with:
+          appCode: ${{ secrets.CHROMATIC_APP_CODE }}
```

Comment:
Note: this secret is manually added in the github settings for the repo (requires the right level of access). There is no way to do automatic key rotation with Chromatic (you need to email and get them to manually change the API key if you want to rotate it)